highfidelity/hifi
# Resorting Developer menu organization

Relates to this bug: https://highfidelity.manuscript.com/f/cases/16869

Organizing the developer menu into a more logical manner.
Adding a Scripting sub menu to hold all the scripting related options and dialogs.

Testing: 
1. Turn on Settings > Developer Menu
2. Review the following menu items to make sure they are placed in their correct spot.
3. Make sure they all function correctly.

Items that were moved from Developer:
Crash > Display Crash Options

UI > Show Overlays

Scripting > Console...
Scripting > API Debugger
Scripting > Entity Script Server Log
Scripting > Verbose Logging
Scripting > Enable Speech Control API
Scripting > Script Log (HMD Friendly)
Scripting > Debug defaultScripts.js
